minorminer/utils/parallel_embeddings.py

```diff
@@ -181,7 +232,12 @@ def find_multiple_embeddings(
         ):
             emb = []
         else:
-            emb = embedder(_S, _T, **embedder_kwargs)
+            if perf_counter() >= timeout:
+                emb = []
+            else:
+                if timeout == 0:
+                    raise ValueError()
+                emb = embedder(_S, _T, **embedder_kwargs)
```

Comment:
I'm assuming that the embedder can return `[]` and thus the break below is required?

Reply:
Should be an empty dictionary (object of length 0) when it fails, added to docstring.